java/src/jmri/managers/AbstractManager.java

```diff
@@ -179,6 +179,14 @@ public void deleteBean(@Nonnull E bean, @Nonnull String property) throws Propert
     public void register(E s) {
         String systemName = s.getSystemName();
 
+        E existingBean = getBeanBySystemName(systemName);
+        if (existingBean != null) {
+            if (s != existingBean) {
+                log.error("systemName is already registered: " + systemName);
```

Comment:
The first String in a log statement is a formatted String, use it as such, since String concatenation is expensive.

Reply:
How should the line be instead?

Reply:
log.error("systemName is already registered: {}",systemName);

Reply:
Fixed.